api/DOMTokenList.json

```diff
@@ -772,10 +772,10 @@
             "description": "<code>force</code> argument",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "25"
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": "25"
```

Comment:
```suggestion
                "version_added": "24"
```

Reply:
There was no chrome_android 24. This broke the CI.

Reply:
Sorry, I was too quick to merge after applying my own suggestion. This is the second time I've done that I think. I'll try to make it a habit, but branch protection would really great too to make it impossible :)

Reply:
cc @ddbeck is branch protection something you can enable?

Reply:
There's some subtleties to overcome, but yes. See #7171 for details. I'll make it my Friday project, or sooner, depending on how long it takes me to do the release today.